inst/shiny/server.R

```diff
@@ -1406,16 +1406,13 @@ shinyServer(function(input, output, session) {
         ResultsName <- gsub(" ", "_", input$ResultsName)
         if (length(myValues$dList) >= 1) {
           if (anyDuplicated(myValues$dList)) {
-            shinyalert::shinyalert("Error", "name already exists. Please use a unique result name",
-                                   type = "error")
+            stop("name already exists. Please use a unique result name")
```

Comment:
@mmkhan19 Quick question (sorry if I've asked it before), why is this alert reverting to a stop from a shinyalert?

Reply:
I changed the shiny alert to a stop for verifying a unique name for the saved results and for some reason the anyDuplicated() wasn't working as expected and I've changed this in the new version.